npm-shrinkwrap.json

```diff
@@ -373,20 +373,17 @@
     "arr-diff": {
       "version": "4.0.0",
       "resolved": "https://registry.npmjs.org/arr-diff/-/arr-diff-4.0.0.tgz",
-      "integrity": "sha1-1kYQdP6/7HHn4VI1dhoyml3HxSA=",
-      "optional": true
```

Comment:
@rfk I think you need to update to latest `npm` 6, otherwise these new fields will be missing and you create the diff in shrinkwrap

Reply:
Whoops, thanks, I don't see why this should need to change `npm-shrinkwrap.json` at all, it must have just regenerated when I reinstalled locally with node 10.